### PR TITLE
Hover and pressed visual updates

### LINKED ISSUE #953: Proposal: Update hover visual
## Summary
Update the hover visual to align more with the depth model. When item is hovered over, the item is lifted. When you get closer to the item, the color becomes lighter.

## Rationale
* The button today indicates hover via grid border lines. Removed the border and using color is more coherent and consistent.
* Follows the other control updates being proposed.

## Important Notes
Here are the scoped changes proposed. Ones with asterisk are places where change is proposed.
image

Controls impacted:

- Button
- ToggleButton
- Checkbox
- SplitButton
- ToggleButton
- ToggleSplitButton

Buttons   Hyperlinks

Buttons   Hyperlinks - Dark Theme

### PR DESCRIPTION
**Pull request by kaiguo**
Closes #953 and #839 

Updated visuals for
- Button
- Checkbox
- RadioButton
- SplitButton
- ToggleButton
- ToggleSplitButton
- TreeView

62232586-795d0780-b37b-11e9-922c-72441b60fddf
62234006-9cd58180-b37e-11e9-9066-cab8ad101683

### CODE REVIEW

**Comment by mdtauk** [comment]
With my experimenting, I tended to also change the foreground colour of the text to 80% on the pressed state.  I also maintained the "lighten for hover, darken for pressed" idea in Dark Mode also, with 40% Fill for Hover, and 10% Fill for Pressed...

Toggle Button - 17-08-2019

File: dev/CheckBox/TestUI/CheckBoxPage.xaml [hunk 1]
```diff
@@ -11,8 +11,10 @@
 
     <StackPanel>
         <CheckBox Content="Two-state CheckBox"/>
-
         <CheckBox Content="Three-state CheckBox" IsThreeState="True"/>
+        <CheckBox Content="Disabled CheckBox Unchekced" IsEnabled="False"/>
```
Comment (llongley) [inline dev/CheckBox/TestUI/CheckBoxPage.xaml]: Nit: typo - "Unchecked"

File: dev/RadioButtons/RadioButtons_themeresources.xaml [hunk 2]
```diff
@@ -5,13 +5,166 @@
     xmlns:local="using:Microsoft.UI.Xaml.Controls">
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Default">
-            <Thickness x:Key="RadioButtonBorderThemeThickness">1</Thickness>
+            <x:Double x:Key="RadioButtonBorderThemeThickness">1</x:Double>
+            <StaticResource x:Key="RadioButtonForeground" ResourceKey="SystemControlForegroundBaseHighBrush" />
```
Comment (llongley) [inline dev/RadioButtons/RadioButtons_themeresources.xaml]: Possibly a dumb question: I'm seeing a whole lot of resources being added here, but I'm not seeing any place where they're actually used.  Am I missing it?

File: dev/TreeView/TreeViewItem.xaml [hunk 3]
```diff
@@ -118,244 +121,11 @@
                             </Grid.ColumnDefinitions>
                             
                             <Grid Grid.Column="0">
-                                <Grid.Resources>
```
Comment (llongley) [inline dev/TreeView/TreeViewItem.xaml]: Just so I'm sure I understand: TreeView is just using the stock check box now?  Why did it previously have its own check box style?

File: dev/RadioButtons/RadioButtons_themeresources.xaml (hunk 2 above)
Comment (kaiguo) [reply dev/RadioButtons/RadioButtons_themeresources.xaml]: Yeah not all of them are used, I thought the pattern was bringing in everything altogether once the control gets touched? But yeah, I can certainly delete the unused once if we don't want them.

File: dev/TreeView/TreeViewItem.xaml (hunk 3 above)
Comment (kaiguo) [reply dev/TreeView/TreeViewItem.xaml]: Yeah that was by design, TreeView had its own CheckBox style, if you look closely at the CheckBox in TreeView today, the background and stroke colors are different from the default CheckBox.

But now we do want a consistent style so I switched to the stock CheckBox.

Comment (llongley) [reply dev/TreeView/TreeViewItem.xaml]: Gotcha.  Thanks!

File: dev/RadioButtons/RadioButtons_themeresources.xaml (hunk 2 above)
Comment (llongley) [reply dev/RadioButtons/RadioButtons_themeresources.xaml]: Are these used elsewhere in the OS repo?  I would've thought that bringing everything over would also involve bringing over the places where they're being used.

Comment (kaiguo) [reply dev/RadioButtons/RadioButtons_themeresources.xaml]: Oh I see, I should've moved over the templates too, I thought they were already in RadioButton.xaml but looks like they are not.

Comment (kaiguo) [reply dev/RadioButtons/RadioButtons_themeresources.xaml]: Added now :)

Comment (llongley) [reply dev/RadioButtons/RadioButtons_themeresources.xaml]: Great, thanks!

**Pull request merged**